include/model.php

```diff
@@ -550,7 +550,10 @@ public function count_posts( $lang, $q = array() ) {
 			$groupby = ' GROUP BY pll_tr.term_taxonomy_id';
 
 			if ( ! empty( $q['m'] ) ) {
```

Comment:
While at it, I think we should make sure that `$q['m']` is a string.
```suggestion
			if ( ! empty( $q['m'] ) && is_string( $q['m'] ) ) {
```